panedrlite/panedrlite/__init__.py

```diff
@@ -4,4 +4,7 @@
 __version__ = pbr.version.VersionInfo('panedrlite').release_string()
 del pbr
 
-from .panedr import edr_to_df, edr_to_dict, read_edr
+from .panedr import ENX_VERSION, \
+    edr_to_df, edr_to_dict, read_edr    # noqa: E402
+
+__all__ = ['ENX_VERSION', 'edr_to_df', 'edr_to_dict', 'read_edr']
```

Comment:
This could maybe use a comment to say that ENX_VERSION is only needed for the tests and not useful for users